### DISCUSSION TITLE
volunteer tester

### DISCUSSION THREAD

**Discussion by okleadfoot** [Courseplay/courseplay#7664] [Q&A] [answered]
I would like to become a tester for this program...I am a disabled Trucker w/Harvest crew experience as well as some farming experience and enjoy this game immensely...I have been playing FS since FS09.
                                                           okleadfoot@hotmail.com

                   yes..ok stands for Okleadfoot.I am just an old trucker:)

**Comment by Tensuko** [comment, marked as answer]
Why not just download https://github.com/Courseplay/Courseplay_FS22/releases

Repository for Issues https://github.com/Courseplay/Courseplay_FS22/issues
